src/Avalonia.Base/Input/InputElement.cs

```diff
@@ -388,6 +414,25 @@ public bool IsEnabled
             get { return GetValue(CursorProperty); }
             set { SetValue(CursorProperty, value); }
         }
+
+        /// <summary>
+        /// Gets or sets a value that determines whether the Holding event can originate
+        /// from that element.
+        /// </summary>
+        public bool IsHoldingEnabled
+        {
+            get { return GetValue(IsHoldingEnabledProperty); }
```

Comment:
These two property definitions can be defined in Gestures class (as it is used there). AddOwner can be used in InputElement, but honestly, I don't think we need these two properties in the InputElement class, as it probably will be very rarely used.